src/ape/managers/project.py

```diff
@@ -112,13 +112,12 @@ def file_needs_compiling(source: Path) -> bool:
                 return True  # New file added
 
             cached = cached_sources[path]
-            cached.compute_checksum(algorithm="md5")
+            cached.calculate_checksum()
```

Comment:
I think this returns the checksum now or something (as opposed to setting it)?

Reply:
yes this

Reply:
Yeah, I'll go ahead and close this PR and open a pinning PR in its place.